staff/carlos-diestro/app-template/news.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>News</title>
+</head>
+<body>
+  <header>
+    <h1>Skylab</h1>
+  </header>
+  <nav>
+    <ul>
+      <li><a href="./home.html">home</a></li>
+      <li><a href="./news.html">news</a></li>
+      <li><a href="./profile.html">profile</a></li>
+      <li><a href="./about.html">about</a></li>
+      <li><a href="./contact.html">contact</a></li>
+      <li><a href="./login.html">login</a></li>
+      <li><a href="./register.html">register</a></li>
+    </ul>
+  </nav>
+  <main>
+    <article>
+      <h3>Lorem ipsum, dolor sit amet consectetur adipisicing elit</h3>
```

Comment:
Why not using a header for news?

Reply:
I had not thought of it, I'll think about it